tests/Tokenizer/TransformersTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Tokenizer;
+
+use PhpCsFixer\Test\AccessibleObject;
+use PhpCsFixer\Tokenizer\Transformers;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Tokenizer\Transformers
+ */
+final class TransformersTest extends TestCase
+{
+    public function testCreateOneTime()
+    {
+        $transformers = Transformers::create();
+        $this->assertSame($transformers, Transformers::create());
+    }
+
+    public function testTransform()
+    {
+        $transformers = Transformers::create();
+        $accessible = new AccessibleObject($transformers);
+        $transformers = $accessible->items;
+
+        $this->assertInternalType('array', $transformers);
+        $this->assertContainsOnlyInstancesOf('PhpCsFixer\Tokenizer\TransformerInterface', $transformers);
+
+        $classes = [];
+        /** @var \SplFileInfo $file */
+        foreach (Finder::create()->files()->in(__DIR__.'/../../src/Tokenizer/Transformer') as $file) {
```

Comment:
isn't it just copy-paste of transformer logic ?
the generation of tested and expected values shouldn't be the same.
use hardcoded expected value (`[Transformer\Foo::Class, TransformerBar::class, ...]` or sth)